misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsMetrics.kt

```diff
@@ -69,8 +69,22 @@ internal class SqsMetrics @Inject internal constructor(metrics: Metrics) {
   )
 
   val sqsDeleteTime = metrics.histogram(
-    "jobs_sqs_delete_latency",
-    "the round trip time to delete messages from SQS",
-    listOf("queueName", "QueueName")
+      "jobs_sqs_delete_latency",
+      "the round trip time to delete messages from SQS",
+      listOf("queueName", "QueueName")
+  )
+
+  val sqsApproxNumberOfMessages = metrics.gauge(
+      "ApproximateNumberOfMessagesVisible",
+      "the approximate number of messages available for retrieval from SQS",
+      listOf("queueName", "QueueName")
```

Comment:
I think we'll also need to include the dimensions that cloudwatch is setting. `namespace` and `stat`